templates/Section.js

```diff
@@ -41,13 +41,17 @@ function generate() {
   const prompt = require("./prompt");
 
   prompt(questions, answers => {
+    const name = answers[0];
```

Comment:
Any reason for moving these out as constants?
Just to prevent duplication farther down the file?

Reply:
No reason other than clarity.